emcc.py

```diff
@@ -1311,6 +1312,9 @@ def default_setting(name, new_default):
     for key in settings_map:
        if key not in COMPILE_TIME_SETTINGS:
          diagnostics.warning('unused-command-line-argument', "linker setting ignored during compilation: '%s'" % key)
+    for arg in state.orig_args:
+        if arg in LINK_ONLY_FLAGS:
```

Comment:
Indentation looks wrong here.

Reply:
I think indentation is also wrong on line 1313, which might be confusing here. It should be 2 spaces everywhere.

Reply:
Sorry I keep messing up the indents :(
Let me fix my vim config. Is there a formatter that the project uses?

Reply:
I also fixed the indent on 1313.

Reply:
We have a `.style.yapf` file at the top level so you can try using `yapf`.. I think it works like clang-format in that you can just format the changes that are part of the your diff.

Reply:
Cool, note that yapf doesn't seem to support f-string https://github.com/google/yapf/issues/451 so it crashes when formatting say, test-core.py